docker/manage

```diff
@@ -205,7 +205,7 @@ configureEnvironment() {
   export AGENT_WALLET_SEED="identity_kit_agent_0000000000000"
   export AGENT_ADMIN_PORT=8024
   export AGENT_WEBHOOK_PORT=8020
-  export AGENT_WEBHOOK_URL=${WEBHOOK_URL:-http://$DOCKERHOST:$IDENTITY_CONTROLLER_HOOK_PORT}
+  export AGENT_WEBHOOK_URL="http://identity-controller-dev:5000"
```

Comment:
Rather than hard-coding the value:
- set the `WEBHOOK_URL` to the desired value before running the manage script. E.g.: `WEBHOOK_URL=something ./manage start`
- if you use the correct port, `DOCKERHOST` should work

The webhook url should also be specific, rather than pointing to a generic API endpoint